docs/pai-management/doc/single-box.md

```diff
@@ -0,0 +1,72 @@
+<!--
+  Copyright (c) Microsoft Corporation
+  All rights reserved.
+
+  MIT License
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+  documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+  the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+  to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+  The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+  BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+  DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
+
+## Customized deploy 
+
+### Index
+- [Step 1. Prepare Deployment Environment](#c-step-1)
+- [Step 2. Prepare configuration](#c-step-2)
+- [Step 3. Boot up Kubernetes](#c-step-3)
+- [Step 4. Push cluster configuration into kubernetes, and set cluster-id](#c-step-4)
+- [Step 5. Start all OpenPAI services](#c-step-5)
+- [Step 6. Check](#c-step-6)
+
+***
+
+
+### Step 1. Prepare Deployment Environment <a name="c-step-1"></a>
+
+
+- [A Guide to install dependency on local host](./how-to-install-depdencey.md)
+
+***
+
+### Step 2. Prepare Configuraiton <a name="c-step-2"></a>
+
+- [Option A. Generate the Cluster Configuration from template](#opt_a)
```

Comment:
It's a good start from generated content, even user want to write their own version. So one option is enough, and use reference to explain each value in configuration.

Reply:
Add some suggestion here, the option A is highly recommended.